src/components/molecules/SecretPasswordForm/SecretPasswordForm.jsx

```diff
@@ -67,9 +68,6 @@ const SecretPasswordForm = ({ buttonText = "Join the party" }) => {
         />
         <div className="form-group">
           {message && <small>{message}</small>}
-          {invalidPassword && (
-            <small className="error-message">Wrong password!</small>
-          )}
           {error && <small className="error-message">An error occured</small>}
```

Comment:
[NB] It looks like `message` probably won't have the same `error-message` styling. Yet we're seemingly using it for errors now. Is that a concern?

Reply:
`error-message` doesn't exist in the .scss and this was showing plain text.

Reply:
Makes sense.

[NB] Does it exist in the `global.scss` or anywhere like that at all? Might be useful to fix the styles here to whatever general 'error style' we use on our other forms.